Harvest4usMOBILE/.idea/gradle.xml

```diff
@@ -13,7 +13,6 @@
             <option value="$PROJECT_DIR$/app" />
           </set>
         </option>
-        <option name="resolveModulePerSourceSet" value="false" />
```

Comment:
This line was re added when I built the app, so not sure if it should be deleted here. Otherwise everything works